scripts/globals/zone.lua

```diff
@@ -1,3 +1,80 @@
+------------------------------------
+--
+-- Contains global functions and variables
+-- related to area specific things
+--
+------------------------------------
+
+
+------------------------------------
+-- zone related IDs
+------------------------------------
+
+    -- Zone Types
+    NONE        = 0
+    CITY        = 1
+    OUTDOORS    = 2
+    DUNGEON     = 3
+    BATTLEFIELD = 4
+    DYNAMIS     = 5
+    INSTANCED   = 6
```

Comment:
Its not required that they perfectly match whats in the source header file. Only that the use be constant in all lua script. I don't understand why the souce is usinging a prefix all those enums, when what they are is plain as day without any prefixing of the individual things
```cpp
enum ZONETYPE
{
    ZONETYPE_stuff = number,
}
```
is there an actual purpose in those prefixes? I normally like my lua stuff to match my core stuff...but these ones annoy me the same way a comment net to the enum telling me "enums zonetypes" would.